README.md

```diff
@@ -1,10 +1,45 @@
 # Rask [![Build Status](https://jenkins.kobert.dev/buildStatus/icon?job=Ratatosk)](https://jenkins.kobert.dev/job/Ratatosk/)
 
-### Motivation
-You might have heard of a game called Nidhogg. The aim of this project is to provide a similar game accessible through the web.
+## Motivation
+
+You might have heard of a game called [Nidhogg](http://nidhogggame.com/). The aim of this
```

Comment:
we might want to link to our wiki page instead, to keep them on our site

Reply:
As I've seen, you already fixed this, thanks a lot! :)